Dockerfile.kafkaInit

```diff
@@ -0,0 +1,59 @@
+FROM openjdk:11-jdk-slim as builder
+
+RUN mkdir -p /code/java-sdk
+WORKDIR /code/java-sdk
+
+COPY commons /code/commons
+COPY specifications /code/specifications
+COPY java-sdk/gradle /code/java-sdk/gradle
+COPY java-sdk/build.gradle java-sdk/settings.gradle  java-sdk/gradlew /code/java-sdk/
+
+ENV GRADLE_OPTS -Dorg.gradle.daemon=false
+
+RUN ./gradlew tasks
+COPY java-sdk/radar-schemas-commons/build.gradle /code/java-sdk/radar-schemas-commons/
+COPY java-sdk/radar-schemas-commons/src /code/java-sdk/radar-schemas-commons/src
+RUN ./gradlew :radar-schemas-commons:jar
+COPY java-sdk/radar-schemas-tools/build.gradle /code/java-sdk/radar-schemas-tools/
+COPY java-sdk/radar-schemas-tools/src /code/java-sdk/radar-schemas-tools/src
+RUN ./gradlew distTar && cd radar-schemas-tools/build/distributions && tar xzf radar-schemas-tools*.tar.gz
+
+
+FROM openjdk:8-alpine
+
+ENV KAFKA_SCHEMA_REGISTRY=http://schema-registry-1:8081
+ENV KAFKA_NUM_PARTITIONS=3
+ENV KAFKA_NUM_REPLICATION=2
```

Comment:
Can we change this to 3? Although it is overridden in docker-compose in RADAR-Docker to 3 still will be nice to have it here as well.

Reply:
Do you mean the replication?

Reply:
Yes. Thank you